### PR TITLE
Promote #69 to Tier 2

### PR DESCRIPTION
**Pull request by dabrady**
We released #69 to Tier 1 on Thursday, and things are looking good. This PR is to promote it to Tier 2 for continued evaluation.

**NOTE:** This PR contains changes that were apparently released to Tier 1 but were not propagated through the rest. Thankfully, these are simple README changes and some changes to a test file, so not concerning.

### CODE REVIEW

File: test/testcafe/test-snippet-prod.js [hunk 1]
```diff
@@ -1,32 +1,28 @@
...
-    }('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
-/* eslint-disable */
+!function(name,path,ctx){
+  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
+}('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
 
-    ProdPerfectKeen.ready(function() {
+ProdPerfectKeen.ready(() => {
+  const client = new ProdPerfectKeen({
```
[the remark this answers is not in the document]
Comment (dabrady) [reply test/testcafe/test-snippet-prod.js:6]: Too bad, and yes it is.

File: test/testcafe/test-snippet-prod.js [hunk 2]
```diff
@@ -1,32 +1,28 @@
...
-    }('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
-/* eslint-disable */
+!function(name,path,ctx){
+  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
+}('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
 
-    ProdPerfectKeen.ready(function() {
+ProdPerfectKeen.ready(() => {
```
[the remark this answers is not in the document]
Comment (dabrady) [reply test/testcafe/test-snippet-prod.js:5]: Too bad, and yes it is.

File: test/testcafe/test-snippet-prod.js [hunk 3]
```diff
@@ -1,32 +1,28 @@
...
-    }('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
-/* eslint-disable */
+!function(name,path,ctx){
+  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
+}('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
```
[the remark this answers is not in the document]
Comment (dabrady) [reply test/testcafe/test-snippet-prod.js:3]: Closing, won't fix

File: test/testcafe/test-snippet-prod.js [hunk 4]
```diff
@@ -1,32 +1,28 @@
...
-      var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
-    }('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
-/* eslint-disable */
+!function(name,path,ctx){
+  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
```
[the remark this answers is not in the document]
Comment (dabrady) [reply test/testcafe/test-snippet-prod.js:2]: Closing, won't fix.

File: test/testcafe/test-snippet-prod.js [hunk 5]
```diff
@@ -1,32 +1,28 @@
-/* eslint-disable */
-    !function(name,path,ctx){
-      var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
-    }('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
-/* eslint-disable */
+!function(name,path,ctx){
```
[the remark this answers is not in the document]
Comment (dabrady) [reply test/testcafe/test-snippet-prod.js:1]: Closing, won't fix.

File: test/testcafe/test-snippet-prod.js [hunk 6]
```diff
@@ -1,32 +1,28 @@
-/* eslint-disable */
```
Comment (dabrady) [inline test/testcafe/test-snippet-prod.js:1]: @rhymiz This file was changed in #61, which you signed off on last year. I'm pretty sure it's just used for testing and will be fine to merge; is that correct?

**Review by rhymiz — APPROVED** [review]

**Comment by dabrady** [comment]
Summarizing a convo with @rhymiz: the thing to do here is to override the Github checks and admin-merge the thing so that it does the thing, and then add tickets to the backlog to work on aligning the project CI config with the branch promotion strategy we use here.

**Comment by dabrady** [comment]
The Semaphore build and deploy associated with this merge can be found here:
https://semaphoreci.com/prodperfect/prodperfect-keen-tracking-js/branches/canary-tier-2/builds/18

**Pull request merged**